components/admin_console/billing/plan_details_jsx_pieces.tsx

```diff
@@ -0,0 +1,354 @@
+                                        <FormattedMarkdownMessage
+                                            id='admin.billing.subscription.planDetails.prolongedOverages'
+                                            defaultMessage='Prolonged overages may result in additional charges.'
+                                        />
+                                        {howBillingWorksLink}
+                                    </div>
+                                </Tooltip>
+                            )}
+                        >
+                            <i className='icon-information-outline'/>
+                        </OverlayTrigger>
+                    }
+                </div>
+            </div>
+            <div className='PlanDetails__subscriptionDate'>
+                <FormattedMessage
+                    id='admin.billing.subscription.planDetails.startDate'
+                    defaultMessage='Start Date: '
+                />
+                <FormattedDate
+                    value={startDate}
+                    day='numeric'
+                    month={getMonthLong(locale)}
+                    year='numeric'
+                />
+            </div>
+            <div className='PlanDetails__subscriptionDate'>
+                <FormattedMessage
+                    id='admin.billing.subscription.planDetails.endDate'
+                    defaultMessage='End Date: '
+                />
+                <FormattedDate
+                    value={endDate}
+                    day='numeric'
+                    month={getMonthLong(locale)}
+                    year='numeric'
+                />
+            </div>
+        </div>
+    );
+};
+
+export const planDetailsTopElements = (userCount: number, isPaidTier: boolean, isPaidTierWithFreeTrial: boolean, userLimit: number) => {
+    const typeSubscription = isPaidTierWithFreeTrial ? 'FREE_TRIAL' : 'MATTERMOST_CLOUD';
+    let userCountDisplay = (
+        <div className='PlanDetails__userCount'>
+            <FormattedMarkdownMessage
+                id='admin.billing.subscription.planDetails.userCount'
+                defaultMessage='{userCount} users'
+                values={{userCount}}
+            />
+        </div>
+    );
+
+    if (!isPaidTier) {
+        userCountDisplay = (
+            <div
+                className={classNames('PlanDetails__userCount', {
+                    withinLimit: (userLimit - userCount) <= 5,
+                    overLimit: userCount > userLimit,
+                })}
+            >
+                <FormattedMarkdownMessage
+                    id='admin.billing.subscription.planDetails.userCountWithLimit'
+                    defaultMessage='{userCount} / {userLimit} users'
+                    values={{userCount, userLimit}}
+                />
+            </div>
+        );
+    }
+    let productName;
+
+    switch (typeSubscription) {
+    case 'FREE_TRIAL':
```

Comment:
This shouldn't be based on the condition of `Free Trial` - we currently only have `Cloud Professional` in trial or not, so we can remove this condition and always show this product. 

We can reintroduce the case when we support multiple products.